src/azure-cli/azure/cli/command_modules/advisor/custom.py

```diff
@@ -132,7 +131,7 @@ def _parse_recommendation_uri(recommendation_uri):
 def _generate_recommendations(client):
     from msrestazure.azure_exceptions import CloudError
 
-    response = sdk_no_wait(True, client.generate)
+    response = client.generate(raw=True)
```

Comment:
so. Is `sdk_not_wait` useless?

Reply:
No, sdk_not_wait is still useful for LRO as it will set "polling" to false if no_wait is true.
But here it should not use sdk_no_wait as it's not a LRO (I checked the SDK method.)
'raw' is another story which is different from LRO, so it should not be set in sdk_no_wait.

All other places I replace 'sdk_no_wait' with 'raw=True' actually are bug fix for those modules.

---
In reply to: [406657030](https://github.com/Azure/azure-cli/pull/12952#discussion_r406657030) [](ancestors = 406657030)